src/manual/Installing/Installing-from-source-Unix.md

```diff
@@ -53,9 +53,9 @@ Version 0.105+: install the Rust toolchain. The best option is to [install the R
 apk add cargo rust
 ```
 
-### Redhat / Centos / Fedora
+### AlmaLinux / Fedora
 
-*For RHEL 8 or Centos Stream*, you will probably need to run this to enable EPEL & PowerTools.
```

Comment:
Same as above, no need to remove Redhat. IDK about mentioning Centos Stream.  

I don't recall if you need to install EPEL and PowerTools for Fedora.  Did you try this?